spec/mailers/alert_mailer_spec.rb

```diff
@@ -66,14 +65,16 @@
 
       it 'shows vacancies' do
         expect(mail.subject).to eq(
-          I18n.t(
-            'job_alerts.alert.email.daily.subject.many',
-            reference: subscription.reference,
-          ),
+          I18n.t('job_alerts.alert.email.subject'),
         )
         expect(mail.to).to eq([subscription.email])
 
         expect(body).to match(/\[#{first_vacancy_presenter.job_title}\]/)
+        expect(body).to match(
+          /2 new jobs matching your search criteria were listed in the last day./,
+        )
+        expect(body).to match(/Subject: #{subscription.subject}/)
```

Comment:
I think `subscription.subject` returns `nil` - correct?

Reply:
ah yes i think you are right, so the test is a false positive